src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java

```diff
@@ -0,0 +1,19 @@
+package se.bjurr.violations.lib.example;
+
+public class CopyOfMyClass {
+ public static String CONstant = "yes";
+
+ void npe(String a, String b) {
+  if (a == null) {
+   System.out.println();
```

Comment:
Reporter: FINDBUGS
Rule: NP_NULL_ON_SOME_PATH
Severity: ERROR
File: se/bjurr/violations/lib/example/CopyOfMyClass.java L8
 Source: se.bjurr.violations.lib.example.CopyOfMyClass


Possible null pointer dereference


<p> There is a branch of statement that, <em>if executed,</em>  guarantees that
a null value will be dereferenced, which
would generate a <code>NullPointerException</code> when the code is executed.
Of course, the problem might be that the branch or statement is infeasible and that
the null pointer exception can't ever be executed; deciding that is beyond the ability of FindBugs.
</p>


<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>